tcms/report/forms.py

```diff
@@ -77,9 +77,6 @@ class CustomSearchDetailsForm(CustomSearchForm):
                               'generating this report.',
         })
 
-    def clean_build_run__product_version(self):
```

Comment:
Why remove this method? Where is it used ?

Reply:
Testing the form with and without this method showed no visible difference.